apps/stripe/src/app/api/stripe/webhook/resolved-webhook-events.ts

```diff
@@ -1,51 +1,27 @@
-import { TransactionActionEnum, TransactionEventTypeEnum } from "@/generated/graphql";
-import { SaleorMoney } from "@/modules/saleor/saleor-money";
+import { AppResult } from "@/modules/app-result/types";
+import { SaleorTransationId } from "@/modules/saleor/saleor-transaction-id";
 import { TransactionEventReportInput } from "@/modules/saleor/transaction-event-reporter";
-import { StripePaymentIntentId } from "@/modules/stripe/stripe-payment-intent-id";
 
-type SharedProps = {
-  pspRef: StripePaymentIntentId;
-  amount: SaleorMoney;
-  date: Date;
-  saleorTransactionId: string;
-};
-
-export abstract class SaleorTransactionEventBase {
-  abstract readonly saleorEventType: TransactionEventTypeEnum;
-  abstract readonly availableActions: TransactionActionEnum[];
-  abstract readonly message: string;
-  readonly saleorTransactionId: string;
-  readonly amount: SaleorMoney;
-  readonly pspRef: StripePaymentIntentId;
+export class TransactionResult {
```

Comment:
I think
1. TransactionResult should be domain specific (your AppResult)
2. This is now "dumb" and holds almost no business stuff other than resolving variables, so you can just name it like what is does. Like `createTransactionEventReportVariablesFromTransactionResult` or ` class TransactionResultVariablesMaker`
3. I don't think it belongs to `api/stripe` anymore (maybe modules/saleor)

Reply:
1 & 2 done in https://github.com/saleor/apps/pull/1885/commits/dbc20052bba0de0331479f43bd84ebd26f476270
3. This is currently used only when handling Stripe webhook so I think it should stay under `app/api/stripe/webhook` etc (the same as we have `usecase-response` files next to Saleor webhook routes